shelley/chain-and-ledger/formal-spec/updates.tex

```diff
@@ -0,0 +1,22 @@
+\section{Software Updates}
+\label{sec:updates}
+
+This section describes how software updates are performed.
+The process of upgrading the system a new version consists of:
+
+\begin{enumerate}
+  \item New software is ready for downloading. The core nodes propose changing the 
+    \textit{software version} in the protocol parameters. If accepted, the proposal
+    will change the protocol parameters at the start of the next epoch.
+  \item All nodes see a new software version and (hopefully) update their software.
```

Comment:
I don't know if we should say "hopefully" here ...

Reply:
yea, good point, I'll remove it.